package.json

```diff
@@ -50,7 +52,10 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "check-lint": "prettier --list-different '**/*.js' && eslint '**/*.js'",
-    "lint": "prettier --write '**/*.js' && eslint --fix '**/*.js'"
+    "lint": "prettier --write '**/*.js' && eslint --fix '**/*.js'",
+    "lint": "npm run lint:js && npm run lint:css",
+    "lint:js": "eslint '**/*.{js}'",
+    "lint:css": "stylelint '**/*.{scss}'"
```

Comment:
```suggestion
    "lint:css": "stylelint '**/*.scss'"
```

Reply:
I guess we don't need these curly brackets — they were from another example where we had to check two file extensions, so it said {css,vue}.